Dockerfile.assisted-installer-controller

```diff
@@ -19,7 +17,14 @@ RUN TARGETPLATFORM=$TARGETPLATFORM make controller
 
 FROM quay.io/centos/centos:stream9
 
-COPY --from=cli-artifacts /usr/share/openshift/assisted/oc /usr/bin/oc
+ENV OCP_VERSION=4.15-el8-beta
+RUN echo -e "\
+[mirror.openshift.com_pub_openshift-v4_multiarch_dependencies_rpms_${OCP_VERSION}_]\n\
+name=created by dnf config-manager from https://mirror.openshift.com/pub/openshift-v4/\$basearch/dependencies/rpms/${OCP_VERSION}/\n\
+baseurl=https://mirror.openshift.com/pub/openshift-v4/\$basearch/dependencies/rpms/${OCP_VERSION}/\n\
+enabled=1" >  /etc/yum.repos.d/mirror.openshift.com_pub_openshift-v4_${TARGETPLATFORM}_dependencies_rpms_${OCP_VERSION}_.repo &&\
+    dnf install --nogpgcheck -y openshift-clients && dnf clean all && rm -rf /var/cache
```

Comment:
@rccrdpccl I see that currently it cannot find `https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rpms/4.15-el8-beta/repodata/repomd.xml` and that's the reason it fails

I guess we have two other options:
* Copy the static binaries instead of rpm installation, e.g. https://mirror.openshift.com/pub/openshift-v4/ppc64le/clients/ocp/4.15.0-rc.0/
* Use an older version. I see that this one is available for ppc: https://mirror.openshift.com/pub/openshift-v4/ppc64le/dependencies/rpms/4.12-el8-beta/

Reply:
Thanks for the hint. I would avoid using binary if possible as it cannot be scanned by security scans, but I'll it take into account